x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_notifications.cy.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createRuleAssetSavedObject } from '../../helpers/rules';
+import {
+  LOAD_PREBUILT_RULES_BTN,
+  LOAD_PREBUILT_RULES_ON_PAGE_HEADER_BTN,
+  UPDATE_PREBUILT_RULES_CALLOUT,
+} from '../../screens/alerts_detection_rules';
+import { deleteFirstRule, waitForRulesTableToBeLoaded } from '../../tasks/alerts_detection_rules';
+import {
+  installAvailableRules,
+  createNewRuleAsset,
+  preventPrebuiltRulesPackageInstallation,
+} from '../../tasks/api_calls/prebuilt_rules';
+import {
+  cleanKibana,
+  resetRulesTableState,
+  deleteAlertsAndRules,
+  reload,
+} from '../../tasks/common';
+import { esArchiverResetKibana } from '../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../tasks/login';
+import { SECURITY_DETECTIONS_RULES_URL } from '../../urls/navigation';
+
+describe('Detection rules, Prebuilt Rules Installation and Update workflow', () => {
+  before(() => {
+    cleanKibana();
+  });
+  beforeEach(() => {
+    login();
+    /* Make sure persisted rules table state is cleared */
+    resetRulesTableState();
+    deleteAlertsAndRules();
+    esArchiverResetKibana();
+    preventPrebuiltRulesPackageInstallation();
+    createNewRuleAsset({
+      rule: createRuleAssetSavedObject({
+        name: 'Test rule 1',
+        rule_id: 'rule_1',
+      }),
+    });
+  });
+
+  describe('Rules installation notification when no rules have been installed', () => {
+    beforeEach(() => {
+      visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+      waitForRulesTableToBeLoaded();
```

Comment:
The above method is waiting already for the page to be loaded, so we can probably remove this one.

Reply:
True, thanks